graphene/sys/versions.txt

```diff
@@ -1,2 +1,6 @@
+<<<<<<< HEAD
 Generated by gir (https://github.com/gtk-rs/gir @ 3918b7c)
+=======
```

Comment:
You probably didn't want this :)

Reply:
I intend to remove this commit and regen once again. :)